rstudio/Dockerfile

```diff
@@ -4,7 +4,7 @@ FROM r-base:latest
 MAINTAINER "Carl Boettiger and Dirk Eddelbuettel" rocker-maintainers@eddelbuettel.com
 
 ## Add RStudio binaries to PATH
-ENV PATH /usr/lib/rstudio-server/bin/:$PATH 
+ENV PATH /usr/lib/rstudio-server/bin/:$PATH
```

Comment:
sorry about this edit, I have rstudio configure to trim trailing white space; let me know if you want this line reverted